e2e/cypress/integration/messaging/collapse_link_spec.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Link preview - Removing it from my view removes it from other user\'s view', () => {
+    before(() => {
+        // # Set the configuration on Link Previews
+        setSystemConsoleLinkPreview();
+        setAccountSettingsLinkPreview('user-1');
+        setAccountSettingsLinkPreview('sysadmin');
```

Comment:
Instead of going thru UI flow (which is not the main test purpose), please add new custom API command (with default value of true) so this can be done like:
```javascript
cy.apiLogin('sysadmin');
cy.apiSaveLinkPreviewPreference();

cy.apiLogin('user-1');
cy.apiSaveLinkPreviewPreference();
```
Please see [cy.apiSaveTeammateNameDisplayPreference](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/api_commands.js#L466) for pattern.

I don't think we have documentation on what to expect on the payload other than inspect the source code and/or network request.  In network request, you'll find something like in the screenshot below.  Ping me anytime if you need help.
<img width="1042" alt="Screen Shot 2019-11-11 at 9 32 17 PM" src="https://user-images.githubusercontent.com/5334504/68591298-5d651800-04cb-11ea-8e4f-f65c49117761.png">